pkg/client/helmclassic/helmclassic.go

```diff
@@ -109,7 +109,15 @@ func (r *Repo) GetDownloadURL(n string, v string) (string, error) {
 	if err != nil {
 		return "", errors.Annotatef(err, "getting %s-%s from index file", n, v)
 	}
-	return chart.URLs[0], nil
+	u, err := url.Parse(chart.URLs[0])
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	chartURL := u.String()
+	if u.Host == "" {
+		chartURL = fmt.Sprintf("%s/%s", r.url, u.String())
+	}
+	return chartURL, nil
```

Comment:
this chunk of logic deserves its own function, so you can add unit tests to it!

Reply:
just created a function in the utils package with specific unit tests